.golangci.yml

```diff
@@ -36,7 +36,16 @@ linters-settings:
     check-shadowing: true
   misspell:
     locale: US
-    ignore-words:
+    ignore-words: []
+  gomodguard:
```

Comment:
You also need to add `gomodguard` between line 15 and 16

Reply:
Thanks, I have enabled gomodguard.